recipes/python-pypi-mirror/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 montag451
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
```

Comment:
As upstream is active, maybe you can ask them to include the license in the source distribution. But nothing blocking.